201-asev2-ilb-with-web-app/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {},
+  "variables": {
+    "existingVirtualNetworkName": "vnet",
+    "existingSubnetName": "subnet-1"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Network/virtualNetworks",
+      "name": "[variables('existingVirtualNetworkName')]",
+      "apiVersion": "2016-03-30",
+      "location": "West Europe",
+      "comments": "Virtual network for overall solution",
+      "tags": {
+        "displayName": "Virtual Network"
+      },
+      "dependsOn": [],
+      "properties": {
+        "addressSpace": {
+          "addressPrefixes": [
+            "10.0.0.0/16"
+          ]
+        },
+        "subnets": [
+          {
+            "name": "[variables('existingSubnetName')]",
+            "properties": {
+              "addressPrefix": "10.0.1.0/24"
+            }
+          }
+        ]
+      }
+    }
+  ],
+  "outputs": {
+    "existingVnetResourceId": {
```

Comment:
output the names of the vnet and subnet, not the ids

Reply:
removed